src/pages/documentation/upgrade-platform-sdks/comparison.md

```diff
@@ -38,9 +38,9 @@ The following tables provide information about the differences between the new E
 | Postbacks - Get/POST URL requests | Yes |	Yes - [Signals](../mobile-core/signal/index.md) extension |
 | Postbacks - PII Get/POST URL requests | Yes | Yes - [Signals](../mobile-core/signal/index.md) extension |
 | Postbacks - Open app deeplink | Yes | Yes - [Signals](../mobile-core/signal/index.md) extension |
-| Push Messaging | Yes | Yes - [Mobile Services](../adobe-analytics-mobile-services/index.md) extension | 
-| In-app Messaging | Yes | Yes - [Mobile Services](../adobe-analytics-mobile-services/index.md) extension |
-| Marketing/Acquisition Links | Yes | Yes - [Mobile Services](../adobe-analytics-mobile-services/index.md) extension |
+| Push Messaging | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/documentation/adobe-analytics-mobile-services/) extension |
+| In-app Messaging | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/documentation/adobe-analytics-mobile-services/) extension |
+| Marketing/Acquisition Links | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/documentation/adobe-analytics-mobile-services/) extension |
```

Comment:
Shouldn't these link to AA - Mobile Services in previous versions instead
https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/